pkg/controller/perconaservermongodbbackup/perconaservermongodbbackup_controller.go

```diff
@@ -181,7 +193,12 @@ func (r *ReconcilePerconaServerMongoDBBackup) Reconcile(ctx context.Context, req
 
 		err = cluster.CheckNSetDefaults(ctx, svr.Platform)
 		if err != nil {
-			return rr, errors.Wrapf(err, "set defaults for %s/%s", cluster.Namespace, cluster.Name)
+			err := errors.Wrapf(err, "wrong PSMDB options for %s/%s", cluster.Namespace, cluster.Name)
```

Comment:
we perform some kind of validations inside the `CheckNSetDefaults `. Maybe we should adjust the message to `invalid cr options used for...`